components/email/src/Email.svelte

```diff
@@ -49,6 +50,10 @@
   import LoadingIcon from "./assets/loading.svg";
   import { downloadFile } from "@commons/connections/files";
   import { DisallowedContentTypes } from "@commons/constants/attachment-content-types";
+  import ReplyIcon from "./assets/reply.svg";
+  import ReplyAllIcon from "./assets/reply-all.svg";
+  import "../../composer/src/Composer.svelte";
+  import type { Message as ComposerMessage } from "@commons/types/Composer";
```

Comment:
I think embedding the composer within the Email component does a few things that I'm not in love with:
1. it moves away from the philosophy of providing devs with interoperable pieces that can respond to one another. For example, getting Availability and Scheduler, or Contact List and Agenda to work nicely with each other is the sort of thing that users will surely ask for; however, because a lot of their use-cases are in isolation, also, we give `dispatch`ed / emitted events and function hooks that developing users can use to build their app the way they want to. In this case, say the user wanted to have their composer reply happen in a fully separate part of their app window? Would be accomplishable with dispatched events (like replyClicked) but not with this method.
2. Increases the file size for all mailbox/email users, regardless of whether they're using replies (uncompressed: mailbox goes from 505kb -> 821kb; email goes from 382kb -> 696kb)